WireAuthentication/WireAuthentication/Sources/WireAuthenticationAssembly.swift

```diff
@@ -0,0 +1,63 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import SwiftUI
+import WireAuthenticationAPI
+internal import WireAuthenticationUI
+internal import WireAuthenticationCore
+
+/// The entry point for the feature module.
+///
+/// To load the contents of the module, create an instance of
+/// the assembly, which you can then use to make required views.
+@MainActor
+public struct WireAuthenticationAssembly {
```

Comment:
Will there be more code than the assembly in the entry point target?
If not, isn't it a bit too much of an overhead to have a separate target?

Reply:
I don't think there would be anything other than instantiating components, wiring them together and returning them. Where would be another place to put it if we don't have a separate target? It couldn't exist inside either the UI or Core targets because the assembly needs to combine components from both.